src/components/questions/helpfulness/Helpfulness.css

```diff
@@ -1,3 +1,7 @@
 .help {
   font-size: small;
+}
+
+.add_link {
+  cursor: default;
```

Comment:
Out of curiosity, why did you add this? Was it a part of the business requirements? Generally you usually want something that is clickable to have the 'pointer' cursor